Interdomain Routing FRR/BGP Transit AS/kathara-lab_bgp-transit-as-redistribute-bgp-non-deterministic_frr/as10r4/etc/frr/frr.conf

```diff
@@ -0,0 +1,17 @@
+! as10r4
+!
+! FRRouting configuration file
+!
+password zebra
+enable password zebra
+!
+log file /var/log/frr/frr.log
+!
+! RIP CONFIGURATION
+!
+router rip
+network eth0
```

Comment:
Please, add indentation to improve readability. (you can find the correct indentation running the command `show running-config` inside `vtysh`)